src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -122,6 +186,24 @@ private bool CanOfferRefactoring(SyntaxNode functionDeclaration, SemanticModel s
             return true;
         }
 
+        protected TParameterSyntax GetParameterNode(SyntaxToken token, int position)
+        {
```

Comment:
Now that we switched to using `TryGetRelevantNodeAsync` to get the selected parameter node, we probably should change this to use that too